public/css/main.css

```diff
@@ -88,11 +88,7 @@ body {
     font-weight: bold;
     color: #35A5B3;
     display: block;
-    transition: 0.3s
-}
-
-.sidenav a:hover, .offcanvas a:focus {
-    color: $f1f1f1;
```

Comment:
This was incorrectly defined. When I fixed the syntax, it was a pretty bad hover color so I'm removing for now.